Automattic/wp-calypso
# GitHub login flow: Add feature flag for GitHub login

Closes https://github.com/Automattic/wp-calypso/issues/86941

## Proposed Changes

This PR adds the `login/github` feature flag.

## Testing Instructions

1. Apply this PR
2. In an incognito window, visit http://calypso.localhost:3000/log-in & http://calypso.localhost:3000/start/account/user-social

You should be presented with placeholder content:

![CleanShot 2024-01-29 at 11 28 32@2x](https://github.com/Automattic/wp-calypso/assets/528287/9e39cda9-a7d0-473c-8a32-1e17b031a40e)
![CleanShot 2024-01-29 at 11 28 27@2x](https://github.com/Automattic/wp-calypso/assets/528287/eb503417-3bbc-4b6f-be63-2e9780e1c2b2)

Retry on the Calypslo Live environment, the placeholder shouldn't be present there.

## Pre-merge Checklist
